index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Code Reviewer Training</title>
+  </head>
+  <body>
+    <h1>Hello Microverse</h1>
```

Comment:
- [ ] I think you can include a <main></main> tag for better use of semantic tags.